tsconfig.json

```diff
@@ -1,18 +1,21 @@
 {
```

Comment:
@NSeydoux @chelseapinka Maybe for later, but we really need to clean up these files and make them consistent across all our repos. I think we'll be able to have a base `tsconfig` in `typescript-sdk-tools` and `extends` it, which should set all the compiler options across all our repos, and then we're just setting files to include in the build.

Reply:
Example of this: https://github.com/mightyiam/tsconfigs